src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -125,6 +127,7 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
             IEnumerable<KeyValuePair<string, ReportDiagnostic>> specificDiagnosticOptions,
             bool concurrentBuild,
             bool extendedCustomDebugInformation,
+            bool debugPlusMode,
```

Comment:
All I could do here is really - "enables debugPlus mode, see codegen for details". That seems too obvious though.